src/pages/Page.vue

```diff
@@ -22,40 +22,36 @@
 
 <script>
     import backgroundImage from '../assets/bg.png';
-    import SearchPanel from '../components/SearchPanel.vue'
-    import SortPanel from '../components/SortPanel.vue'
-    import Movies from '../components/Movies.vue'
+    import SearchPanel from '../components/SearchPanel.vue';
+    import SortPanel from '../components/SortPanel.vue';
+    import Movies from '../components/Movies.vue';
+    import { mapState } from 'vuex';
     export default {
         name: 'Page',
         components: {
             SearchPanel,
             SortPanel,
             Movies
         },
-        props: {
-            movies: Array
-        },
         data: function () {
             return {
                 backgroundImage: backgroundImage,
-                searchType: 'name',
-                searchText: '',
-                sortType: 'year'
             }
         },
         methods: {
             onSearch: function (search) {
                 if (search.isType) {
-                    this.searchType = search.searchType;
+                    this.$store.commit('APPLY_SEARCH_TYPE', search.searchType);
```

Comment:
1) It would be good to use actions, because later when you will be asking API return movies it going to be async action, all mutations are sync
2) it would be good to use mapActions instead of direct call on $store